lib/algorithm.ts

```diff
@@ -353,3 +354,45 @@ function hazing({ tweet, sentiment }: TweetData): Rank {
     score: 0,
   }
 }
+
+/**
+ * U.S. politics is so awesome, and everybody should love to read about it,
+ * no matter their background, interests, or country of residence.
+ * You WILL form an opinion on us!
+ */
+function usPolitics({ tweet, sentiment }: TweetData): Rank {
+  // List of hot topics, power posters, and coronavirus conspiracy
+  const topics = [
+    "🇺🇸", "american?", "patriot", "president", "states", "psyop",
+    "conservatives?", "democrat", "demonrat", "republican", "crt", "woke\\w*",
+    "wokeness", "abortion", "pro-life", "authoritarian", "drag", "ideology",
```

Comment:
pro-life is a term that is being used when discussing abortion, so I would avoid boosting tweets containing it

Reply:
It's all Americans: https://twitter.com/search?q=pro-life&src=typed_query&f=top